cristin-organization/src/main/java/no/unit/nva/cristin/organization/DefaultOrgFetchProvider.java

```diff
@@ -0,0 +1,18 @@
+package no.unit.nva.cristin.organization;
+
+import java.util.Map;
+import no.unit.nva.model.Organization;
+import nva.commons.apigateway.RequestInfo;
+
+public class DefaultOrgFetchProvider implements IClientProvider
+                                                    <RequestInfo, IFetchApiClient<Map<String, String>, Organization>> {
+
+    @Override
+    public IFetchApiClient<Map<String, String>, Organization> getClient(RequestInfo params) {
```

Comment:
Should i assume we always use RequestInfo as param so we dont have to specify type whenever we use interface IClientProvider?

Reply:
It is not used for now it seems. Personally, I would avoid letting RequestInfo leak into any code but the handler itself.

Reply:
Here is where you define your T,  Alternatively you could set it to String apiVersion, as that is what you need to know.